### DISCUSSION TITLE
[Feature request]: FSR for PJ64

### DISCUSSION THREAD

**Discussion by elcorsico** [project64/project64#2326] [Ideas]
### Request box

hi dev
possible to Add FSR shader for PJ64 ? plugin ?

### Reason box

good and nice graphiq

### Examples

games are very beautiful with FSR technology

**Comment by project64** [comment]
if it goes in to GlideN64 then it will be automatically pulled in to project64. 

If you want in Project64's video plugin, not even going to happen for a long time unless some new gfx programmer wants to add it
